sdk/template/azure-template-stress/templates/job.yaml

```diff
@@ -0,0 +1,54 @@
+{{- include "stress-test-addons.deploy-job-template.from-pod" (list . "stress.java-template") -}}
+{{- define "stress.java-template" -}}
+metadata:
+  labels:
+    testName: "{{ .Release.Name }}"
+spec:
+  containers:
+    # simple and fast .NET Core HTTP server to run tests against
+    # When writing real stress test, you probably won't need it and would use corresponding Azure Service.
+    - name: server
+      image: stresspgs7b6dif73rup6.azurecr.io/stress/simplehttpserver
```

Comment:
Is it safe to assume this will always be replaced? If not I should add a `registry` value to the helm context so we don't have to take a dependency on what is a cluster-specific ACR.

Reply:
Yes, this container is not needed for real SDK tests and only exists for demo purposes. At some point we probably will add azure-core tests where we'd need HHTP server, but  would probably use storage with SAS connection string instead.